vmdb/app/models/brand.rb

```diff
@@ -0,0 +1,62 @@
+class Brand
+  attr_accessor :customer_name
+  attr_accessor :vmdb_name
+  attr_accessor :custom_logo
+
+  def initialize(attrs = {})
+    attrs.each do |n, v|
+      public_send("#{n}=", v)
+    end
+  end
+
+  def exist?
+    customer_name && vmdb_name && custom_logo
+  end
+
+  def class_text
+    custom_logo ? "whitelabeled" : ""
+  end
```

Comment:
I could extract `"whitelabel"`

```ruby
def if_custom(txt)
  custom_logo && txt
end

haml:
  current_brand.if_custom("whitelabel")
```

Don't want to over think this one. I'm not sure if we even need the css around `whitelabel`. We may need to move that stuff into the haml. (Not my call, and not the time to figure this out)